src/standalone/Vegetation/Canopy.jl

```diff
@@ -636,8 +636,8 @@ function canopy_surface_fluxes(
     model::CanopyModel,
     Y,
     p,
-    t::FT,
-) where {FT}
+    t,
+) where {FT <: AbstractFloat}
```

Comment:
also here: https://github.com/CliMA/ClimaLSM.jl/blob/ad31773009c9309d5e45b083603873a8914be286/src/shared_utilities/drivers.jl#L27, no need for <: AbstractFloat